zdx/reader.go

```diff
@@ -1,241 +1,43 @@
 package zdx
 
 import (
-	"io"
 	"os"
+
+	"github.com/brimsec/zq/zio/bzngio"
+	"github.com/brimsec/zq/zng/resolver"
 )
 
-type reader struct {
-	filename string
-	file     *os.File
-	frame    []byte
-	valsize  int
-}
+const (
+	FrameSize = 32 * 1024
+)
```

Comment:
Do you affirmatively want this to be public?

Reply:
I will leave it for now.  We probably want to make this stuff configurable soon and this will become the default.